def/flow.js

```diff
@@ -164,13 +168,23 @@ def("Identifier")
 def("TypeParameterDeclaration")
   .bases("Node")
   .build("params")
-  .field("params", [def("Identifier")]);
+  .field("params", [or(def("Identifier"), def("TypeParameter"))]);
 
 def("TypeParameterInstantiation")
   .bases("Node")
   .build("params")
   .field("params", [def("Type")]);
 
+def("TypeParameter")
+  .bases("Identifier")
```

Comment:
In Flow, this node is not a subtype of Identifier, but is its own node with a name, location, variance, and bound.